tests/test_cli.py

```diff
@@ -0,0 +1,13 @@
+"""Tests for functions implemented in cli.py."""
+
+import click
+from unittest.mock import patch
+from click.testing import CliRunner
+from check_in.cli import cli
+
+@patch("check_in.cli.GithubAPI")
+def test_cli_method(mocker):
+    runner = CliRunner()
+    mocker.return_value = "test"
```

Comment:
This probably does not work, because you didn't add pytest-mock to requirements (unless you installed it manually)